NOTICE

```diff
@@ -4,3 +4,8 @@ Copyright 2017 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+----------------------------------------------------------------------------------------------------
+
+Circe CRC library
```

Comment:
isn't Circe is under [Apache license](https://github.com/trevorr/circe/blob/master/LICENSE) only?